### PR TITLE
Move customizing metadata tutorial to permanent doc location

### PR DESCRIPTION
**Pull request by elrayle**

### CODE REVIEW

File: pages/hydra/developer_resources/customize_metadata/hyrax_1.0.0.rc1/controlled-vocabulary.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Prereq: Defining a Controlled Vocabulary"
+keywords: Customize Metadata
+categories: How to Do All the Things
+permalink: customize-metadata-controlled-vocabulary.html
+folder: hydra/how-to/customize_metadata/hyrax_1.0.0.rc1/controlled-vocabulary.md
```
Comment (jcoyne) [inline pages/hydra/developer_resources/customize_metadata/hyrax_1.0.0.rc1/controlled-vocabulary.md]: Do we want to encode rc1 here or would Hyrax 1.0 be sufficient?

Comment (bess) [inline pages/hydra/developer_resources/customize_metadata/hyrax_1.0.0.rc1/controlled-vocabulary.md]: +1 for Hyrax 1.0

Comment (elrayle) [inline pages/hydra/developer_resources/customize_metadata/hyrax_1.0.0.rc1/controlled-vocabulary.md]: I will make this hyrax_1.0.  

Broader discussion in Issue #37

**Comment by jenlindner** [comment]
Replaced occurrences of 1.0.0.rc1 with 1.0.

**Comment by elrayle** [comment]
@jcoyne Good catch on the need to update the metadata.  @jenlindner Thanks for taking care of making the change.

**Pull request merged**